starwars/src/App.js

```diff
@@ -1,41 +1,91 @@
-import React, { Component } from 'react';
-import './App.css';
+import React from 'react';
+import CharacterProfile from './components/CharacterProfile';
 
-class App extends Component {
+class App extends React.Component {
   constructor() {
     super();
     this.state = {
-      starwarsChars: []
+      starwarsChars: [],
+      nextPage: '',
+      previousPage: '',
+      currentPage: `https://swapi.co/api/people/?page=1`
```

Comment:
So this was how I wrote my state to get pagination to work, as well. However, I don't think you actually need this line for it to work. 
```suggestion
     
```